### ISSUE TITLE
[Suggestion] Play once

### ISSUE THREAD

**Issue by SteaSteaStea** [SekaiRed/AmbienceBlock#5]
It would be nice to have an option to play a track only once. Maybe add a redstone "on pulse" option?
Maybe there's a way to do this already but I couldn't figure it out. The only option currently seems to be setting up a timer to turn the redstone off after x seconds but if a sound is for example just 1 second long then it's difficult to have it not loop at least once before being turned off.

Thank you for the amazing mod by the way.

**Comment by SekaiRed** [comment]
I thought about implementing this but the problem is, it is pretty much the behavior of a command block with a `/playsound` command, I thought it would be redundant to do something you can achieve in vanilla.

Thank you!

**Comment by SteaSteaStea** [comment]
Well I am not sure how other people use this mod. I use it in survival to add ambience and sound effects to different machines/teleporters/doors etc. 

One of the things I've made with this mod is a vocoded warning message when my storage is running out of space which repeats once every 30mins when this occurs. I've basically managed to make it all work with the mod as is by making the duration of the redstone signal as long as the audio file using pulse repeaters and latches and so on. I guess in this case it would be just a matter of convenience but without having all the extra modded redstone repeaters and so on it would be probably way too difficult to be adjusting signal lengths with single tick precision.
